go.mod

```diff
@@ -0,0 +1,78 @@
+module github.com/joshuatcasey/libdependency
```

Comment:
```suggestion
module github.com/paketo-buildpacks/libdependency
```

Reply:
Yeah - we'll have to update all of these references. I've left them alone for now so that the included unit / integration tests can run. When all other issues on the PR are resolved I'll add a commit to do this rename.

Reply:
bump

Reply:
@sophiewigmore do you feel that there are no other blocking issues with the PR? I'll push the renames if you do.